tests/testnets/simple.common/01-gen-configs.sh

```diff
@@ -149,10 +148,12 @@ datadir = \${sbws_home}/datadir
 sbws_home = $(pwd)
 
 [tor]
-control_type = socket
-control_location = \${paths:sbws_home}/$scanner_tor_socks_proxy_nick/control_socket
-socks_host = $scanner_tor_socks_proxy_ip
-socks_port = $socksport
+extra_lines =
+    TestingTorNetwork 1
+    NumCPUs 1
```

Comment:
Why NumCPUs 1?
Seems like an odd option.

Reply:
There's ~12 Tor processes that are run at once in these tests and I thought I would attempt to not peg all of everyone's CPU cores, if possible.